docs/node/getting-started.md

```diff
@@ -68,32 +68,35 @@ service ElizaService {
 }
 ```
 
-## Generate code
+## Generate code using Buf v2
 
-We're going to generate our code using [Buf](https://www.npmjs.com/package/@bufbuild/buf),
-a modern replacement for Google's protobuf compiler. We installed Buf earlier,
-but we also need a configuration file to get going. (If you'd prefer, you can
-skip this section and use `protoc` instead &mdash; `protoc-gen-connect-es`
-behaves like any other plugin.)
+We're going to generate our code using [Buf][buf], a modern replacement for
+Google's protobuf compiler. We installed Buf earlier, but we also need a few
+configuration file to get going. (If you'd prefer, you can skip this section
+and use `protoc` instead &mdash; `protoc-gen-connect-es` behaves like any other
+plugin.)
 
-First, tell Buf how to generate code with a `buf.gen.yaml` file:
+First, scaffold a basic [`buf.yaml`][buf.yaml] by running `npx buf config init`. Next,
+tell Buf how to generate code by putting this into
+[`buf.gen.yaml`][buf.gen.yaml]:
```

Comment:
Updated the explanation for the Node.js example to align with those of other languages.